Sources/ARKit-CoreLocation/Extensions/CLLocation+Extensions.swift

```diff
@@ -14,6 +14,14 @@ public struct LocationTranslation {
     public var latitudeTranslation: Double
     public var longitudeTranslation: Double
     public var altitudeTranslation: Double
+
+    // needed because the "free" init function is marked fileprivate by default, so LocationTranslation isn't
```

Comment:
I'd love to see this rewritten as a doc comment for someone who will be using the `LocationTranslation` struct. Does it matter that the free init function is marked `fileprivate` to them?